x/crosschain/keeper/grpc_query_cctx_rate_limit.go

```diff
@@ -90,15 +93,46 @@ func (k Keeper) ListPendingCctxWithinRateLimit(c context.Context, req *types.Que
 		return uint32(len(cctxs)) >= limit
```

Comment:
minor note about this closure: 
I'd much prefer not capturing cctx by reference. Capture by value is much clear and less magic, and there is no performance loss as it's slice. 
```
func(cctxs) bool {
		// #nosec G701 len always positive
		return uint32(len(cctxs)) >= limit
```

Reply:
already resolved